index.md

```diff
@@ -84,6 +90,3 @@ What broad approaches do we follow when we do this thing?
 What specific bits of software (commercial or open source) do
 we use to help us do this thing?
 ```
-
-The service manual has some useful information on
-[learning about and writing user needs](https://www.gov.uk/service-manual/user-research/start-by-learning-user-needs).
```

Comment:
@joelgsamuel Did you mean to remove this?

Reply:
@SteveMarshall it is there (https://github.com/ministryofjustice/technical-guidance/blob/eaad04ea3c84f01b47b8e895dac25e5efe0d53d4/index.md) I think its just perhaps how I did the edits in the browser (I need to stop using github.com directly via a browser...)